addons/account/views/report_invoice.xml

```diff
@@ -197,6 +197,72 @@
                             </div>
                         </div>
                     </div>
+                    <div t-if="o.invoice_payment_term_id">
+                        <p t-if="o.invoice_payment_term_id.has_early_payment">
+                            <strong>
+                                Early Payment (before <span t-esc="o.invoice_payment_term_id._get_last_date_for_discount(o.invoice_date)"/>):
+                                <span t-esc="o.currency_id.symbol"/>
+                                <span t-esc="o.invoice_early_pay_amount_after_discount"/>
```

Comment:
I'm not sure it's necessary to have all that stuff as computed not-stored fields. You could call a method here to retrieve then directly. To be discussed